gravitee-apim-console-webui/src/services-ngx/api-v2.service.ts

```diff
@@ -154,14 +151,16 @@ export class ApiV2Service {
   }
 
   search(searchQuery?: ApiSearchQuery, sortBy?: ApiSortByParam, page = 1, perPage = 10, manageOnly = true): Observable<ApisResponse> {
```

Comment:
I would recommend this as a fix as it touches a little bit less of the code and may have less averse effects 🤔 
```suggestion
  search(searchQuery: ApiSearchQuery = {}, sortBy?: ApiSortByParam, page = 1, perPage = 10, manageOnly = true): Observable<ApisResponse> {
```